### PR TITLE
Continuous backup

### PR DESCRIPTION
**Pull request by almirsarajcic**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ straightforward as possible.
 ### Changed
 - UI: audio button moved into paperclip menu
 - `Chat.Db.BackupDbSupervisor` now receives the name of the DB it needs to supervise as an argument
+- Continuous backup
```
Comment (sergey-lukianov) [inline CHANGELOG.md:19]: merge with `main` please
cause this line is likely to be merged into previous release

File: lib/chat/db/write_queue.ex [hunk 2]
```diff
@@ -95,24 +99,27 @@ defmodule Chat.Db.WriteQueue do
     |> noreply()
   end
 
-  def handle_cast({:mirror, sink}, state) do
+  def handle_cast({:mirrors, sink}, state) do
     state
-    |> q_state(mirror: sink)
+    |> q_state(mirrors: sink)
```
Comment (sergey-lukianov) [inline lib/chat/db/write_queue.ex:104]: rewriting previous mirror?

File: mix.exs [hunk 3]
```diff
@@ -13,7 +13,7 @@ defmodule Chat.MixProject do
       deps: deps(),
       test_coverage: [
         ignore_modules: [ChatWeb.ChannelCase, ChatWeb.Gettext, ChatWeb.Router],
-        summary: [threshold: 62]
+        summary: [threshold: 61]
```
Comment (sergey-lukianov) [inline mix.exs:16]: we never decrease threshold :)

Comment (almirsarajcic) [reply mix.exs:16]: Haha tried to sneak this one in :D
I'll see what I can test for a quick win.

File: lib/chat/db/write_queue.ex (hunk 2 above)
Comment (almirsarajcic) [reply lib/chat/db/write_queue.ex:104]: What do you mean? The only difference is now the mirror is a list of writers.

Comment (almirsarajcic) [reply lib/chat/db/write_queue.ex:104]: Fixing this.

Comment (almirsarajcic) [reply lib/chat/db/write_queue.ex:104]: Actually, it seems to work as expected. For InternalDB and BackupDB, mirrors get set to `nil`, and for MainDB it gets set to `[Chat.Db.InternalDb.Writer, Chat.Db.BackupDb.Writer]`.

Comment (salseeg) [reply lib/chat/db/write_queue.ex:104]: It is odd that it works
May we have a test case for this?

Comment (almirsarajcic) [reply lib/chat/db/write_queue.ex:104]: Added tests for main and backup sync in platform's PR.

**Review by sergey-lukianov — APPROVED** [review]

**Pull request merged**